client/src/ui/PageFooter.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+import Link from "next/link";
+import Image from "next/image";
+import "./styling/PageFooter.css";
+import { Typography } from "@mui/material";
+
+const PageFooter = () => {
+    return (
+        <footer className="footer">
+            <div className="footer-container">
```

Comment:
uncessary div since the footer already wraps around the container

Reply:
It has been fixed!